app/models/format_review_file.rb

```diff
@@ -5,7 +5,6 @@ class FormatReviewFile < ApplicationRecord
 
   belongs_to :submission
 
-  validates :submission_id, :asset, presence: true
   validates :asset_cache, presence: true, if: proc { |f| !f.submission.nil? && f.submission.author_edit && f.submission.status_behavior.collecting_format_review_files? }
```

Comment:
After thinking about this some more, and reviewing some of the documentation for CarrierWave, I think what we actually want to do is remove:

`validates :asset_cache, presence: true, if: proc { |f| !f.submission.nil? && f.submission.author_edit && f.submission.status_behavior.collecting_format_review_files? }`

instead of

`validates :submission_id, :asset, presence: true`

It seems like the duplication in the error messages was actually being caused by the `:asset` validation _and_ the `:asset_cache` validation being present.  I'm not sure why we are validating the `:asset_cache`, since it will ultimately be present if `:asset` is present.  `:asset_cache` isn't really a database field, so it's probably better practice to keep the validation on `:asset` (which is a database field), and remove the validation on `:asset_cache`.